app_dart/test/request_handlers/get_branches_test.dart

```diff
@@ -162,5 +217,41 @@ void main() {
       expect((result.single)['branch']['branch'], 'flutter-branch-created-now');
       expect((result.single)['id'].runtimeType, String);
     });
+
+    test('should always retrieve release branches', () async {
+      expect(db.values.values.whereType<Branch>().length, 1);
+
+      const String id = 'flutter/flutter/$betaBranchName';
```

Comment:
```suggestion
      const String id = '${Config.flutterSlug}/$betaBranchName';
```

Reply:
Sure, updated to flutterSlug.

Umm what is the advantage of using a slug over a string?

Reply:
It minimizes the work for any LSC changes to a monorepo

Reply:
Thanks for explaining!